docs/config.example.yaml

```diff
@@ -1,27 +1,26 @@
-
 ################################################################################
 ##                             Network interface                              ##
 ################################################################################
 server:
-  # Address to which we bind
-  address: ""      # default value
-  # Port on which we bind
-  port:    3836    # default value
+  # Address to which we bind (default: 127.0.0.1)
+  address: "0.0.0.0"
+  # Port on which we bind (default: 3836)
+  port:    4242
 
   # Endpoint of Agora
-  agora_endpoint: http://127.0.0.1:2826
+  agora_endpoint: http://8.8.8.8:4567
```

Comment:
dns.google ?
I wonder why you set `8.8.8.8` as the default value.

Reply:
It's not the default, just an invalid example. I could also use `127.1.1.1`

Reply:
Fixed